api-specificatie/openapi.yaml

```diff
@@ -688,9 +619,11 @@ paths:
         Parameter adresseerbaarobjectidentificatie-query bevat de 16 cijferige identificatie van een ligplaats, standplaats of verblijfsobject"
       parameters:
         - $ref: '#/components/parameters/adresseerbaarobjectidentificatie-query'
+        - $ref: '#/components/parameters/locatie-query'
         - $ref: 'https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/master/api-specificatie/common.yaml#/components/parameters/expand'
         - $ref: 'https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/master/api-specificatie/common.yaml#/components/parameters/fields'
         - $ref: 'https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/master/api-specificatie/common.yaml#/components/parameters/acceptCrs'
+        - $ref: 'https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/master/api-specificatie/common.yaml#/components/parameters/contentCrs'
```

Comment:
Kan/Gaat het voorkomen dat een consumer aangeeft dat hij in de response geometrie in een bepaalde formaat verwacht, maar dat zijn request geometrie in een andere formaat opmaakt?

Reply:
Nee, dat gaat het niet voorkomen en zoals ik dit hoofdstuk in de API-srategie lees is dat zelfs gewenste functionaliteit

Reply:
Is het dan nodig om contentCrs als querystring parameter op te nemen? Is alleen acceptCrs niet voldoende?